app/controllers/notes_controller.rb

```diff
@@ -34,6 +36,8 @@ def show
     @activities = @note.activities.latest
     @subscription = @note.subscription_for(user: current_user)
     load_conflicting_revisions(@note)
+    find_mentionable_users
+    read_item_notifications(@note, current_user)
```

Comment:
can this be, as part of a separate PR, pushed up to the concern?

Reply:
will do

Reply:
https://github.com/dradis/dradis-ce/pull/377